pkg/buses/kafka/dispatcher/dispatcher.go

```diff
@@ -167,7 +167,7 @@ func (d *dispatcher) subscribe(subscription *channelsv1alpha1.Subscription, para
 				glog.Infof("Dispatching a message for subscription %s/%s: %s -> %s", subscription.Namespace,
 					subscription.Name, subscription.Spec.Channel, subscription.Spec.Subscriber)
 				message := fromKafkaMessage(msg)
-				err := d.messageDispatcher.DispatchMessage(subscription.Spec.Subscriber, subscription.Namespace, message)
+				err := d.messageDispatcher.DispatchMessage(subscription.Spec.Subscriber, subscription.Spec.ReplyTo, subscription.Namespace, message)
```

Comment:
These feels like we should wrap the message and replyTo in a struct and then DispatchMessage is (subscriber, event)?

Reply:
Should we consider whether `subscriber` and `namespace` also belong in the struct? (would be a single "envelope" message parameter if so).

WDYT @scothis? I addressed all of the other comments (so far) besides this one.

Reply:
The message and destination feel fundamental to me. I'd be in favor of something like:

```
func (d *MessageDispatcher) DispatchMessage(message *Message, destination string, opts *DispatchOpts) error
```

Reply:
I went with a `DispatchDefaults` struct. RFAL @n3wscott @scothis. Thanks!